docs/create-packages/Symbol-Packages-snupkg.md

```diff
@@ -21,28 +21,36 @@ ms.reviewer:
 
 # Creating symbol packages (.snupkg)
 
+Symbol packages allow you to improve the debugging experience of your NuGet packages.
+
 ## Prerequisites
 
 [nuget.exe v4.9.0 or above](https://www.nuget.org/downloads) or [dotnet.exe v2.2.0 or above](https://www.microsoft.com/net/download/dotnet-core/2.2), which implement the required [NuGet protocols](../api/nuget-protocols.md).
 
 ## Creating a symbol package
 
-A snupkg symbol package can be created from a .nuspec file or from a .csproj file. NuGet.exe and dotnet.exe are both supported. When the options ```-Symbols -SymbolPackageFormat snupkg``` are used on the nuget.exe pack command a .snupkg file will be created in addition to the .nupkg file.
+You can create a snupkg symbol package using dotnet.exe, NuGet.exe, or MSBuild. If you're using NuGet.exe, you can use the following commands to create a .snupkg file in addition to the .nupkg file:
```

Comment:
snupkg is an extension. Perhaps put it in parenthesis like 
symbol package (`.snupkg`)

Reply:
There are two types of symbol packages, .snupkg and .symbols.nupkg. The intent here is to say "a symbol package in the snupkg format". Is there a better name than "snupkg" to refer to the snupkg format?

I feel that saying "symbol package (.snupkg)" is strange as it makes it seem like all symbol packages have the `.snupkg` format. Does that make sense?